docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_OpenAPI.adoc

```diff
@@ -67,8 +115,8 @@ The MicroProfile OpenAPI specification defines an HTTP endpoint that serves an O
 The OpenAPI endpoint is registered using the configured path (e.g. http://localhost:8080/openapi) local to the root of the host associated with a given deployment.
 
 [NOTE]
-Currently, the OpenAPI endpoint for a given virtual host can only document a single JAX-RS deployment.
-To use OpenAPI with multiple JAX-RS deployments registered with different context paths on the same virtual host, each deployment should use a distinct endpoint path.
+When the OpenAPI endpoint for a given host documents _a single_ JAX-RS deployment, the title and description of the OpenAPI document are obtained from the <display-name/> and <description/> elements, respectively, from the web application deployment descriptor (e.g. web.xml), if otherwise unspecified.
+When the OpenAPI endpoint for a given host documents _multiple_ JAX-RS deployments, any discordant 'info' or 'externalDocs' OpenAPI document properties will be sourced from the MicroProfile config properties prefixed by 'mp.openapi.extensions._server-name_._host-name_'.
```

Comment:
AIUI these are new MP Config keys which makes this a Feature not an Enhancement.

Is mp.openapi.extensions._server-name_._host-name_ sufficiently namespaced? If smallrye or some other/future thing we integrate uses extension keys we can easily have conflicts between user defined servers names and keys used by our integrated libraries.

Reply:
@pferraro Why not 'mp.openapi.extensions.wildfly'? We use the namespace that we reasonably should control, the same as smallrye does.

I assume that was the design intent at MP OpenAPI.

Reply:
> Why not 'mp.openapi.extensions.wildfly'?

1. The current naming scheme is what I inherited from the prototype integration created by @MikeEdgar
2. The spec suggests that the vendor (i.e. WF) has full control over the "mp.openapi.extensions." prefix:

> Vendors that wish to provide vendor-specific configuration via MP Config (instead of another native configuration framework) must use the prefix mp.openapi.extensions.

I can certainly move the new configuration properties under mp.openapi.extensions.wildfly.*, though that suggests re-locating existing configuration properties (which we can do by suppling default values that look for the old property names), but, AFAIR, MP config does not have a deprecation mechanism for property names.

Reply:
Ok; good point; we are already in this situation, and this is just a matter of not using further initial elements that may conflict with a reasonable server-name. I don't see 'smallrye' as a likely issue.

Reply:
OK, I see your point.  Perhaps we use something like:
mp.openapi.extensions.server._server-name_.host._host-name_.foo.bar